specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-01-01/imds.json

```diff
@@ -890,6 +926,136 @@
         }
       }
+    },
+    "LoadBalancerMetadata": {
+      "type": "object",
+      "description": "This is the response from the Loadbalancer_GetMetadata operation.",
+      "properties": {
+        "publicIpAddresses": {
+          "type": "array",
+          "description": "List of the instance level public or private IP of VM.",
+          "items": {
+            "type": "object",
+            "description": "This contains the publicIpAddress properties.",
+            "$ref": "#/definitions/PublicIpAddress"
+          }
+        },
+        "inboundRules": {
+          "type": "array",
+          "description": "List of load balancing rules or inbound NAT rules using which the Load Balancer directs traffic to the VM. Frontend IP addresses and the Private IP addresses listed here belong to the Load Balancer.",
+          "items": {
+            "type": "object",
+            "description": "This contains the inboundRule properties.",
+            "$ref": "#/definitions/LoadBalancerInboundRule"
+          }
+        },
+        "outboundRules": {
+          "type": "array",
+          "description": "List of outbound rules by which the VM behind Load Balancer sends outbound traffic. Frontend IP addresses and the Private IP addresses listed here belong to the Load Balancer.",
+          "items": {
+            "type": "object",
+            "description": "This contains the outboundRule properties.",
+            "$ref": "#/definitions/LoadBalancerOutboundRule"
+          }
+        },
+        "tunnelInterfaces": {
+          "type": "array",
+          "description": "List of tunnelInterface by which the VM behind Gateway Load Balancer sends or receives traffic to/from consumer Loadbalancer or Internet. This list is only available when VM is in the backend pool of a Gateway Load balancer. Learn more at aka.ms/gatewaylb",
+          "items": {
+            "type": "object",
+            "description": "This contains the tunnelInterface properties.",
+            "$ref": "#/definitions/LoadBalancerTunnelInterface"
+          }
+        }
+      }
+    },
+    "PublicIpAddress": {
+      "type": "object",
+      "description": "Loadbalancer public ipAddress Metadata.",
+      "properties": {
+        "frontendIpAddress": {
+          "type": "string",
+          "description": "This is instance level public ipAddress associated to the NIC on the VM"
+        },
+        "privateIpAddress": {
+          "type": "string",
+          "description": "This is private ipAddress assigned to the NIC on the VM"
+        }
+      }
+    },
+    "LoadBalancerInboundRule": {
+      "type": "object",
+      "description": "This contains the Load Balancer inbound Rule properties.",
+      "properties": {
+        "frontendIpAddress": {
+          "type": "string",
+          "description": "This is frontend ipAddress used in load balancing rules and inbound NAT rules using which the Load Balancer directs traffic to the VM."
```

Comment:
"ipAddress" -> "IP address". Please update all the other occurrences too.

Reply:
i also feel we could remove the "This is" part, since it feels redundant.

Reply:
"using which" -> ", using which". adding a separator here to help customer breaking down the sentence.

there are a lot of similar hits of this, so please update them as well.

Reply:
Adding "This is" is to be same style as other parts.